testing/recordings/TestInteractionCameraHotkeys.log

```diff
@@ -3,6 +3,9 @@ RenderEvent 0 0 0 0 0 0 0
 KeyPressEvent 0 0 0 51 1 3 0
 CharEvent 0 0 0 51 1 3 0
 KeyReleaseEvent 0 0 0 51 1 3 0
+KeyPressEvent 0 0 0 53 1 5 0
```

Comment:
Should we create a new test instead of modifying an existing one?

Reply:
Yes I agree. We should also create separate tests for each camera position (1,3,7,9)

Reply:
I think it makes sense to add to this test suite since this is testing the camera hotkeys. 
However, let's also create another issue to separate each interaction into its own test, as there is value in checking the baseline image after each key is pressed instead of only checking that the last baseline image is correct.

Reply:
I have created a new test only for Orthographic Projection and removed the test case from CameraHotKeys